apps/src/gamelab/GameLabVisualizationColumn.jsx

```diff
@@ -40,13 +40,17 @@ const styles = {
 };
 
 const SongSelector = Radium(class extends React.Component {
+  static propTypes = {
+    setSong: PropTypes.func.isRequired
+  };
+
   render() {
     return (
-      <div id="song_selector">
+      <div>
         <label><b>{gamelabMsg.selectSong()}</b></label>
-        <select style={styles.selectStyle}>
+        <select id="song_selector" style={styles.selectStyle} onChange={this.props.setSong}>
```

Comment:
important: you should specify a `value` property for this `select`, and supply the value that is stored in redux. otherwise, you will have an [uncontrolled component](https://reactjs.org/docs/uncontrolled-components.html), which is bad in this case because it moves the source of truth of which song is selected out of redux.